src/glyphy.h

```diff
@@ -24,6 +24,12 @@
 extern "C" {
 #endif
 
+#if defined (_MSC_VER) && defined (BUILDING_GLYPHY) && !defined (GLYPHY_STATIC)
+# define GLYPHY_API __declspec(dllexport)
```

Comment:
What about dllimport for when linking to glyphy instead of building it?

While you are at it, shouldn't you define GCC symbol visibility in the non-Windows case? This allows for compiler optimizations...

Reply:
Hi @eli-schwartz,

It's actually optional in this case, since we aren't dealing with C++ name mangling and public variables, although it could give a very very small performance advantage (if at all--GLib and HarfBuzz, for instance, does the same thing on this regard).

For the visibility attributes in GCC, I am not confident enough to try to do that here in this PR.

Thanks for the notes.